namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-test-dev/resources/user-filestore.tf

```diff
@@ -0,0 +1,29 @@
+# auto-generated from fb-cloud-platforms-environments
+##################################################
+# User Filestore S3
+module "user-filestore-s3-bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket"
```

Comment:
without a fixed reference like https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/blob/19ed7266742ca67f095ecfa585fd8a0e80d63542/example/main.tf#L16 a future release might delete and recreate the bucket

Reply:
I see. Sorry, I'm new to terraform. is that assumed best practice? I just copied the the example from the README here https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket

Would you like me to update the README there too?

Reply:
argh, we have an example in /example/ and another in the README sorry will correct

Reply:
I didn't spot that, I'll leave that in your capable hands 😄